addons/medical_blood/functions/fnc_onBleeding.sqf

```diff
@@ -20,7 +20,7 @@ params ["_unit"];
 if (((vehicle _unit) != _unit) && {!((vehicle _unit) isKindOf "StaticWeapon")}) exitWith {}; // Don't bleed on ground if mounted
 
 private _lastTime = _unit getVariable [QGVAR(lastTime), -10];
-private _bloodLoss = (if (GVAR(useAceMedical)) then {GET_BLOOD_LOSS(_unit) * 2.5} else {getDammage _unit * 2}) min 6;
+private _bloodLoss = (if (GVAR(useAceMedical)) then {GET_BLOOD_LOSS(_unit) * 2.5} else {getDamage _unit * 2}) min 6;
```

Comment:
This was perfect, [not even an error](https://community.bistudio.com/wiki/getDammage)! :D